composer.json

```diff
@@ -130,6 +130,9 @@
         "allow-plugins": {
             "laminas/laminas-component-installer": true,
             "laminas/laminas-skeleton-installer": true
+        },
+        "platform": {
+            "php": "7.3.99"
```

Comment:
It's because you forgot to update that value to `7.4.99` when you force-pushed this commit. [drop PHP 7.3 ](https://github.com/laminas/laminas-mvc-skeleton/commit/4994114a41599a607a894fe3b46fe4c42c17b940).

Reply:
It will impact users of 8.0, 8.1 and 8.2 when that is out. No go for skeleton app used with `composer create-project`

Reply:
We can likely configure renovate to consider PHP to be at 7.4.x, but then contributors will keep doing a `composer update` which pushes dependencies out of supported ranges.